### PR TITLE
Add implementation for Web Components with Compost mixins

### PR DESCRIPTION
**Pull request by lamplightdev**
Hello,

Here is an implementation of the app using vanilla web components and some simple mixins which I've packaged as [compost](https://github.com/lamplightdev/compost).

Demo app: https://compost-35844.firebaseapp.com

Let me know if you need anything else.

Many thanks.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
```
Comment (housseindjirdeh) [inline package-lock.json]: This looks like it may have been committed by accident. Can we remove this? 🙏

File: site/_contributors/chris-h.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+weight: 31
+name: 'Chris H'
```
Comment (housseindjirdeh) [inline site/_contributors/chris-h.txt]: 🚀 🚀 🚀

File: site/_apps/compost-hn.txt [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+weight: 31
+title: 'Vanilla Web Components with Compost mixins'
+github-title: 'lamplightdev/compost-hn'
+library: 'Compost mixins'
+module-bundling: 'Rollup'
+service-worker: 'Shell + data caching with Workbox'
+perfomance-patterns: 'Local caching of API results, CDN cache'
+server-side-rendering: 'no'
+api: 'Node-hnapi (unofficial)'
```
Comment (housseindjirdeh) [inline site/_apps/compost-hn.txt]: @davideast rolled out a faster API that you can use here with a CDN cache (api.hnpwa.com/v0) so I would definitely consider using that here instead!

Comment (davideast) [reply site/_apps/compost-hn.txt]: Yes! I would love to see the results. Check it out here: https://blog.hnpwa.com/introducing-the-hnpwa-api-656db6b88ec1

**Comment by housseindjirdeh** [comment]
@lamplightdev to help clear our backlog a bit, I'll be closing this for now. But please let me know if you're still interested in exploring adding this submission and we can reopen it!

**Comment by lamplightdev** [comment]
Thanks @housseindjirdeh I will prepare an updated PR later today

**Comment by housseindjirdeh** [comment]
Awesome, thank you 🙏 

Looking forward to landing this in :D

**Comment by lamplightdev** [comment]
Great - I think this is good to go now @housseindjirdeh ?

**Pull request merged**